### DISCUSSION TITLE
Support NPM peer dependencies for a meteor atmosphere package

### DISCUSSION THREAD

**Discussion by vparpoil** [meteor/meteor#13352] [Feature Requests]
When a package need a peer dependency, there is no way to indicate it inside package.js file.
This package used to be a workaround : https://github.com/Meteor-Community-Packages/check-npm-versions but is not compatible Meteor 3 anymore

It would be great if we could indicate a peer dependency that would trigger a warning for the developer if missing, instead of rendering a white page when the dependency is missing.

**Comment by StorytellerCZ** [comment]
Version 2 of check-npm-versions should be compatible with Meteor 3.
